app/views/hosts/show.html.erb

```diff
@@ -28,26 +28,26 @@
       </tbody>
     </table>
 
-    <ul id="myTab" class="nav nav-tabs">
-      <li class="active"><a href="#properties" data-toggle="tab"><%= _('Properties') %></a></li>
-      <li><a href="#metrics" data-toggle="tab"><%= _('Metrics') %></a></li>
+    <ul id="host-show-tabs" class="nav nav-tabs">
+      <li class="<%= active_tab_class('properties') %>"><a href="#properties" data-toggle="tab"><%= _('Properties') %></a></li>
```

Comment:
It would be better to refactor the whole `<li>..</li>` line into a single helper in `hosts_helper.rb`:
``` ruby
def tab_header(id: 'properties', text: _(id), class: [])
  class << 'active' if id == @active_tab

  tag.li(class: class) do
    link_to(text, "##{id}", data: {toggle: 'tab' })
  end
end
```